ibis/backends/tests/test_export.py

```diff
@@ -215,6 +215,22 @@ def test_table_to_parquet(tmp_path, backend, awards_players):
     )
 
 
+def test_table_to_parquet_dir(tmp_path, backend, awards_players):
+    outparquet_dir = tmp_path / "out"
+    # max_ force to write more than one parquet file
+    awards_players.to_parquet_dir(
+        outparquet_dir, max_rows_per_file=3000, max_rows_per_group=3000
+    )
+
+    parquet_files = list(outparquet_dir.glob("*.parquet"))
```

Comment:
Globs are not guaranteed to return in any particular order, so you need to sort them.

```suggestion
    parquet_files = sorted(outparquet_dir.glob("*.parquet"), key=lambda path: int(path.with_suffix("").name.split("-")[1])
```